docs/vectorizer/migrating-from-extension.md

```diff
@@ -0,0 +1,37 @@
+Previous versions of pgai vectorizer used an extension to provide the vectorizer
+functionality.  We have removed the need for the extension and put the
+vectorizer code into the pgai python library. This change allows the vectorizer
+to be used on more PostgreSQL cloud providers (AWS RDS, Supabase, etc.) and
+simplifies the installation and upgrade process.
+
+Versions that used the extension:
+- `ai` extension version < 0.10.0
+- `pgai` python library version < 0.10.0
+
+# Migrating from the extension to the python library
+
+We made this change in a way that will allow current users of the vectorizer to
+continue using the feature without interruption, but they will have to modify how they
+upgrade vectorizer functionality in the future.
+
+The upgrade process is as follows:
+
+1. **Upgrade the extension:** Run ALTER EXTENSION ai UPDATE TO '0.10.0' to detach the vectorizer catalog tables and functions from the extension. This leaves them in your database in the ai schema, and the vectorizer will continue to work.
+2. **Upgrade the pgai python library:** Upgrade the pgai Python library to version `>0.10.0`. This can be done with `pip install -U pgai` or via your `requirements.txt` or similar dependency file.
```

Comment:
```suggestion
2. **Upgrade (or install) the pgai python library:** Install pgai version `>0.10.0`. This can be done with `pip install -U pgai` or via your `requirements.txt` or similar dependency file.
```